src/execution_plan/ops/op_node_by_label_scan.c

```diff
@@ -162,6 +163,15 @@ static OpResult NodeByLabelScanReset(OpBase *ctx) {
 	return OP_OK;
 }
 
+static OpBase *NodeByLabelScanClone(const ExecutionPlan *plan, const OpBase *opBase) {
+	assert(opBase->type == OPType_NODE_BY_LABEL_SCAN || OpType_NODE_BY_LABEL_AND_ID_SCAN);
+	NodeByLabelScan *op = (NodeByLabelScan *)opBase;
+	OpBase *clone = NewNodeByLabelScanOp(plan, op->n);
+	if(opBase->type == OpType_NODE_BY_LABEL_AND_ID_SCAN)
+		NodeByLabelScanOp_SetIDRange((NodeByLabelScan *) clone, op->id_range);
```

Comment:
Wouldn't the cloned operation be optimised to `OpType_NODE_BY_LABEL_AND_ID_SCAN` if possible?